components/bl-mapbox/src/helpers.js

```diff
@@ -188,7 +188,7 @@ export const useMarkers = (markers, mapRef, onMarkerClick) => {
       setMarkersArray([]);
```

Comment:
Change useEffect + useState for useMemo

Ask Pavlo for isDataChanged function or write me, and recreate only new markers. As I see marker has `update` method.

Reply:
Also take a loot at every `new` and ask yourself "is it correct to create new element in this situation or we could update existed?"